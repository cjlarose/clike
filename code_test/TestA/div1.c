

void printchar(char);
void printint(int);

void main()
{
  int x;
  int a, b, c;

  a = 200;
  b = 3;
  c = 4;

  
  x = a/c;
  printint(x);
  printchar(10); // newline

  x = a/b;
  printint(x);
  printchar(10);
  
  x = b/c;
  printint(x);
  printchar(10);
  
}
  
