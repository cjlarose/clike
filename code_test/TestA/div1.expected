Loaded: /usr/local/share/spim/exceptions.s
50
66
0
